app/uk/gov/hmrc/bindingtariffclassification/model/EnumJson.scala

```diff
@@ -20,8 +20,25 @@ import play.api.libs.json._
 
 object EnumJson {
 
-  implicit def format[E <: Enumeration](enum: E): Format[E#Value] = {
+  def format[E <: Enumeration](enum: E): Format[E#Value] = {
     Format(Reads.enumNameReads(enum), Writes.enumNameWrites)
   }
 
+  def readsMap[E, B](implicit erds: Reads[E], brds: Reads[B]): JsValue => JsResult[Map[E, B]] = (js: JsValue) => {
```

Comment:
For some reason `Json.format[...]` doesnt work if you have a Map where the key is anything other than a String so I had to do this...